assets/fonts/fonts.css

```diff
@@ -32,7 +30,6 @@
     font-family: 'Proxima Nova';
     src: url('Proxima Nova Alt Regular-webfont.eot');
     src: url('Proxima Nova Alt Regular-webfont.eot?#iefix') format('embedded-opentype'),
-         url('Proxima Nova Alt Regular-webfont.woff2') format('woff2'),
```

Comment:
This wasn't included in the zip. Not sure what the implications are of not having this.